server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/project/model/Project.java

```diff
@@ -122,6 +122,9 @@ public static class ProjectMember {
         @JsonProperty(access = Access.READ_ONLY)
         private String accountName;
 
+        @JsonProperty(access = Access.READ_ONLY)
+        private Boolean userEnabled;
```

Comment:
It's a bit odd that the project has this `userEnabled` field. How should we understand this?

Reply:
Oh sorry, I didn't see it was added to `ProjectMember` rather than `Project`. Then it's cool to me~